frontend/appflowy_flutter/lib/workspace/presentation/home/menu/view/view_action_type.dart

```diff
@@ -38,12 +38,12 @@ extension ViewMoreActionTypeExtension on ViewMoreActionType {
     }
   }
 
-  Widget icon(Color iconColor, {ViewState? state}) {
+  Widget icon(Color iconColor, {bool? state}) {
     switch (this) {
       case ViewMoreActionType.delete:
         return const FlowySvg(name: 'editor/delete');
       case ViewMoreActionType.toggleFavorite:
-        if (state!.view.isFavorite) {
+        if (state!) {
```

Comment:
```
        return const FlowySvg(name: isFavorited! ? 'home/unfavorite' : 'home/favorite');
```
or 
```
        return const FlowySvg(name: 'home/${isFavorited! ? 'unfavorite' : 'favorite'}');
```
or
```
        return const FlowySvg(name: 'home/${isFavorited! ? 'un' : ''}favorite');
```

Reply:
It's outdated; I have refactored this part.